### PR TITLE
fix(ui): notifications appearing on top in unintended situations

### PR DESCRIPTION
**Pull request by Subjective**
This fixes an issue where notifications were obscuring UI elements of plugins like [Noice](https://github.com/folke/noice.nvim/blob/main/lua/noice/config/views.lua
).

### CODE REVIEW

**Comment by mehalter** [comment]
What UI situations would happen in which you would not want the notification to appear on the top? I am not sure I can think of one

**Comment by mehalter** [comment]
A screenshot might be helpful as well showing the unintended behavior

**Comment by Subjective** [comment]
@mehalter The Noice cmdline. From what I can tell, setting the zindex 150 is low enough that it notifications appear above the cmdline while remaining underneath mason and lazy.
Before:
image
After:
image

**Comment by mehalter** [comment]
@Subjective if you set it to `200` will it show the last one on top? If a notification comes up while the command line is open then that would be over, but if you open the command line with a notification open then it would be on top? I'm not sure how ties are resolved. But I think that might be better behavior out of the box if possible

File: lua/plugins/ui.lua [hunk 1]
```diff
@@ -52,7 +52,7 @@ return {
     init = function() require("astronvim.utils").load_plugin_with_func("nvim-notify", vim, "notify") end,
     opts = {
       on_open = function(win)
-        vim.api.nvim_win_set_config(win, { zindex = 1000 })
+        vim.api.nvim_win_set_config(win, { zindex = 150 })
```
Comment (mehalter) [inline lua/plugins/ui.lua:55]:
```suggestion
        vim.api.nvim_win_set_config(win, { zindex = 175 })
```

Ok, after some testing and playing around, I think 175 is a good default here. I checked the zindex of our default plugins and this seems to be a good default. If other people want different results then they should override this themselves.

**Review by mehalter — CHANGES_REQUESTED** [review]

**Review by mehalter — APPROVED** [review]
Awesome! Thanks for taking the time to open a PR to resolve this :)

**Pull request merged**